application/org.openjdk.jmc.flightrecorder.flameview/src/main/resources/page.template

```diff
@@ -10,15 +10,28 @@
 	<![endif]-->
 </head>
 
-<body onresize="resizeFlameGraph()">
+<body>
```

Comment:
out of curiosity, why the change from using `onresize` to adding a listener to the browser?

Reply:
This was another tricky part, when the swt.browser is created it gets the flameGraph default size. Even when you resize the jmc window the size is not updated. You need to check the event on the Browser object and force the update by as I'm doing with resizeFlameGraph.